laravel-svelte-port/svelte-ui/src/lib/stores/conversations.svelte.ts

```
@@ -45,48 +50,67 @@ class ConversationsStore {
         case 'unread':
           return (b.unreadCount || 0) - (a.unreadCount || 0);
         case 'priority':
-          const priorityOrder: Record<string, number> = { urgent: 0, high: 1, medium: 2, low: 3, '': 4 };
-          return (priorityOrder[a.priority || ''] || 4) - (priorityOrder[b.priority || ''] || 4);
+          const priorityOrder: Record<string, number> = {
+            urgent: 0,
+            high: 1,
+            medium: 2,
+            low: 3,
+            '': 4,
+          };
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)